modules/FormModel.js

```diff
@@ -86,7 +86,7 @@
      */
     constructor: function(attributes, options) {
       options = options || {};
-      this.__cache = {};
+      this.cache = {};
```

Comment:
I made them top level properties so that you can set them via extension instead of just via behaviorOptions.

```
DataBehavior.extend({
  cache: someCacheCollection
});
```